### PR TITLE
Add a link to the wave-in app to the README.md

### PR DESCRIPTION
**Pull request by psh2849**
We changed check-in method from the QR code check-in method to the sound wave check-in method.

### **The features of the app are as follows:**

- Check-in using sound-wave communication.
- Protect user privacy by using a safe number.
- Automatic volume control function.
- Save user's check-in location and time.

###  Github link
https://github.com/euphony-io/check-in-speaker

Co-Authored-By: Sooyeon Jo <35393459+jopopcorn@users.noreply.github.com>
Co-Authored-By: hoho00 <50264056+hoho00@users.noreply.github.com>
Co-Authored-By: HACore <58473522+HACore@users.noreply.github.com>
Co-Authored-By: kmsaidev <88221290+kmsaidev@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,9 +70,10 @@ Web version is also available now. but only transmitter version.
 - [Euphony Speaker](https://github.com/euphony-io/euphony-speaker) : Euphony Demo to generate sound data.
 - [Sound Helper](https://github.com/euphony-io/sound-helper) : Euphony Demo for patients and assistants in case of an emergency.
 - [Soundless Check-in](https://github.com/euphony-io/soundless-check-in) : Euphony Demo to check in through sound wave.
-- [Here Is My Card](https://github.com/euphony-io/here-is-my-card) : Euphony Demo for sharing buisness cards.
+- [Here Is My Card](https://github.com/euphony-io/here-is-my-card) : Euphony Demo for sharing business cards.
```
Comment (yeonns) [inline README.md:73]: good typo fix!

File: README.md [hunk 2]
```diff
@@ -70,9 +70,10 @@ Web version is also available now. but only transmitter version.
...
 - [Looking For Job](https://github.com/euphony-io/looking-for-job) : Euphony Demo to send and receive resumes.
 - [Listen My Order](https://github.com/euphony-io/listen-my-order) : Euphony Demo for communicate restaurant menu information.
+- [Wave-In](https://github.com/euphony-io/check-in-speaker) : Euphony Demo to check-in using a safe Number.
```
Comment (yeonns) [inline README.md:76]: How about fixing N to n of "Number"?

Comment (psh2849) [reply README.md]: Yes I fix now!

**Comment by kuro11pow2** [comment]
Woow! This app looks really good! 😄

File: README.md (hunk 1 above)
Comment (YoungSeokHong) [reply README.md:73]: You're very meticulous! Nice!😀

**Review by YoungSeokHong — COMMENTED** [review]
Great job!
I think my team also need to add Wave-In (for store)

**Comment by aiclaudev** [comment]
Great! Design of this app is so fancy :)

File: README.md [hunk 3]
```diff
@@ -70,9 +70,10 @@ Web version is also available now. but only transmitter version.
 - [Euphony Speaker](https://github.com/euphony-io/euphony-speaker) : Euphony Demo to generate sound data.
 - [Sound Helper](https://github.com/euphony-io/sound-helper) : Euphony Demo for patients and assistants in case of an emergency.
 - [Soundless Check-in](https://github.com/euphony-io/soundless-check-in) : Euphony Demo to check in through sound wave.
-- [Here Is My Card](https://github.com/euphony-io/here-is-my-card) : Euphony Demo for sharing buisness cards.
+- [Here Is My Card](https://github.com/euphony-io/here-is-my-card) : Euphony Demo for sharing business cards.
 - [Looking For Job](https://github.com/euphony-io/looking-for-job) : Euphony Demo to send and receive resumes.
 - [Listen My Order](https://github.com/euphony-io/listen-my-order) : Euphony Demo for communicate restaurant menu information.
+- [Wave-In](https://github.com/euphony-io/check-in-speaker) : Euphony Demo to check-in using a safe number.
```
Comment (designe) [inline README.md:76]:
```suggestion
- [Wave-In](https://github.com/euphony-io/check-in-speaker) : COVID-19 Check-in solution using a safe number based on MVVM model.
```

Comment (psh2849) [reply README.md]: That would be better! I fix it!

File: README.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ Web version is also available now. but only transmitter version.
 - [Here Is My Card](https://github.com/euphony-io/here-is-my-card) : Euphony Demo for sharing business cards.
 - [Looking For Job](https://github.com/euphony-io/looking-for-job) : Euphony Demo to send and receive resumes.
 - [Listen My Order](https://github.com/euphony-io/listen-my-order) : Euphony Demo for communicate restaurant menu information.
-- [Wave-In](https://github.com/euphony-io/check-in-speaker) : Euphony Demo to check-in using a safe number.
+- [Wave-In](https://github.com/euphony-io/wave-in-speaker) : COVID-19 Check-in solution using a safe number based on MVVM model.
```
Comment (jopopcorn) [inline README.md:76]: Great!! 😆

**Pull request merged**